skills/definitions/player/MinimumAttackSpeed.cs

```diff
@@ -0,0 +1,48 @@
+namespace Turbo.plugins.patrick.skills.definitions.player {
+
+	using System;
+	using System.Collections.Generic;
+	using parameters;
+	using parameters.types;
+	using Plugins;
+
+	public class MinimumAttackSpeed : AbstractDefinition {
+		public float minimumAttackSpeed { get; set; }
+
+		public override DefinitionType category => DefinitionType.Player;
+
+		public override string attributes => $"[ minimumAttackSpeed: {minimumAttackSpeed} ]";
+
+		public override List<AbstractParameter> GetParameters(IController hud) {
+			return new List<AbstractParameter> {
+				SimpleParameter<float>.of(nameof(minimumAttackSpeed), x => minimumAttackSpeed = x)

```

Comment:
Float should not be possible as a parameter currently, since the generated UI can only deal with implemented types.
EIther use `int` and recalculate, or add the the support for the `float` template. Problem is it currently can't distinguish wether to use `int` or `float` when calling the setter. I don't know if thats easily possible.
Easier would be to go `int` and do math.
To see what has to be changed, [see this](https://github.com/petikk/pHelperPlugin/blob/a8aeff67af7cc2a0f8970ad6dd6f80c8c4efcb3a/util/winformutil/ControlHelper.cs#L23)